WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -307,7 +307,7 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
                 style: .cancel,
                 handler: nil)
             let disconnect = UIAlertAction(
```

Comment:
I'm going to be really nitpicky, but let's rename the action from `disconnect`, too.